examples/complete/main.tf

```diff
@@ -103,23 +112,6 @@ module "vpc_endpoints" {
       policy          = data.aws_iam_policy_document.dynamodb_endpoint_policy.json
       tags            = { Name = "dynamodb-vpc-endpoint" }
     },
-    ssm = {
```

Comment:
we don't need all of these - they just make the example take longer and cost more